bootstrap/kubeadm/api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -71,13 +71,13 @@ type KubeadmConfigStatus struct {
 	// +optional
 	BootstrapData []byte `json:"bootstrapData,omitempty"`
 
-	// ErrorReason will be set on non-retryable errors
```

Comment:
This would be a breaking change to the cabpk v1alpha2 types. If we want to make this change, we need a new v1alpha3 group plus conversions.